### ISSUE TITLE
[Bug]: Pre-transform error: Failed to resolve import

### ISSUE THREAD

**Issue by yxw007** [modbender/nuxt-tiptap-editor#11]
# Reproduction Step
- git clone git@github.com:modbender/nuxt-tiptap-editor.git
- cd nuxt-tiptap-editor
- pnpm install
- pnpm dev:prepare
- pnpm dev

# Environment
- System : win10
- Node.js: v22.6.0

# Output
```
Administrator@DESKTOP-BR92P8D MINGW64 /e/h5_workspace/nuxt-tiptap-editor (main)
$ pnpm dev:prepare

> nuxt-tiptap-editor@2.0.0 dev:prepare E:\h5_workspace\nuxt-tiptap-editor
> nuxt-module-build build --stub && nuxt-module-build prepare && nuxi prepare playground

ℹ Stubbing nuxt-tiptap-editor                                                                                                               21:13:58        
ℹ Cleaning dist directory: ./dist                                                                                                           21:13:58        
Tiptap Editor initialized
ℹ Compiled plugins/server.mjs in 594.08ms                                                                                                   21:14:15        
ℹ Compiled plugins/client.mjs in 620.54ms                                                                                                   21:14:15        
ℹ Compiled types/plugins.d.ts in 624.26ms                                                                                                   21:14:15        
✔ Types generated in .nuxt                                                                                                                  21:14:15        
ℹ Tiptap Editor initialized                                                                                                                      21:14:23   
ℹ Using default Tailwind CSS file                                                                                               nuxt:tailwindcss 21:14:24   
✔ Types generated in playground/.nuxt                                                                                                            21:14:27

Administrator@DESKTOP-BR92P8D MINGW64 /e/h5_workspace/nuxt-tiptap-editor (main)
$ pnpm dev

> nuxt-tiptap-editor@2.0.0 dev E:\h5_workspace\nuxt-tiptap-editor
> nuxi dev playground

Nuxt 3.13.1 with Nitro 2.9.7                                                                                                                      21:14:33
                                                                                                                                                  21:14:38
  ➜ Local:    http://localhost:3000/
  ➜ Network:  use --host to expose

ℹ Tiptap Editor initialized                                                                                                                      21:14:48
ℹ Using default Tailwind CSS file                                                                                               nuxt:tailwindcss 21:14:48
  ➜ DevTools: press Shift + Alt + D in the browser (v1.4.1)                                                                                       21:14:55


 WARN  Slow module @nuxt/devtools took 6399.61ms to setup.                                                                                        21:14:55  

ℹ Tailwind Viewer: http://localhost:3000/_tailwind/                                                                             nuxt:tailwindcss 21:14:58
✔ Vite client built in 57ms                                                                                                                      21:15:00
✔ Vite server built in 1726ms                                                                                                                    21:15:02
✔ Nuxt Nitro server built in 2144 ms                                                                                                       nitro 21:15:04
ℹ Vite client warmed up in 1ms                                                                                                                   21:15:04
ℹ Vite server warmed up in 1394ms                                                                                                                21:15:06
ℹ ✨ new dependencies optimized: primevue/menubar                                                                                                21:15:09
ℹ ✨ optimized dependencies changed. reloading                                                                                                   21:15:09

 ERROR  Pre-transform error: Failed to resolve import "E:h5_workspace                                                                             21:15:12  
uxt-tiptap-editorsrccustom-extensionsextension-image-upload" from "playground/components/TipTapImage.vue". Does the file exist?


 ERROR  Internal server error: Failed to resolve import "E:h5_workspace                                                                           21:15:12  
uxt-tiptap-editorsrccustom-extensionsextension-image-upload" from "playground/components/TipTapImage.vue". Does the file exist?
  Plugin: vite:import-analysis
  File: E:/h5_workspace/nuxt-tiptap-editor/playground/components/TipTapImage.vue:6:95
  4  |  import { StarterKit as TiptapStarterKit } from '@tiptap/starter-kit';
  5  |  import { Image as TiptapImage } from '@tiptap/extension-image';
  6  |  import { ImageUpload as TiptapImageUpload, ImagePlaceholder as TiptapImagePlaceholder } from 'E:\h5_workspace\nuxt-tiptap-editor\src\custom-extensions\extension-image-upload';
     |                                                                                                ^
  7  |  /* Injection by vite-plugin-vue-inspector Start */
  8  |  import { createElementVNode as __createElementVNode,createElementBlock as __createElementBlock,createVNode as __createVNode } from 'vue'
      at TransformPluginContext._formatError (file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite@5.4.3_@types+node@22.5.4_sass@1.78.0_terser@5.31.6/node_modules/vite/dist/node/chunks/dep-BaOMuo4I.js:49192:41)
      at TransformPluginContext.error (file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite@5.4.3_@types+node@22.5.4_sass@1.78.0_terser@5.31.6/node_modules/vite/dist/node/chunks/dep-BaOMuo4I.js:49187:16)
      at normalizeUrl (file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite@5.4.3_@types+node@22.5.4_sass@1.78.0_terser@5.31.6/node_modules/vite/dist/node/chunks/dep-BaOMuo4I.js:63970:23)
te/dist/node/chunks/dep-BaOMuo4I.js:63970:23)
      at async file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite@5.4.3_@types+node@22.5.4_sass@1.78.0_terser@5.31.6/node_modules/vite/dist/node/chunks/dep-BaOMuo4I.js:64102:39
      at async Promise.all (index 5)
      at async TransformPluginContext.transform (file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite@5.4.3_@types+node@22.5.4_sass@1.78.0_terser@5.31.6/node_modules/vite/dist/node/chunks/dep-BaOMuo4I.js:64029:7)
      at async file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite-plugin-inspect@0.8.7_@nuxt+kit@3.13.1_magicast@0.3.5_rollup@4.21.2__rollup@4.21.2_vite@5_p6m2mgxqdfrcrveyul5azkmpku/node_modules/vite-plugin-inspect/dist/index.mjs:401:17
      at async PluginContainer.transform (file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite@5.4.3_@types+node@22.5.4_sass@1.78.0_terser@5.31.6/node_modules/vite/dist/node/chunks/dep-BaOMuo4I.js:49033:18)
      at async loadAndTransform (file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite@5.4.3_@types+node@22.5.4_sass@1.78.0_terser@5.31.6/node_modules/vite/dist/node/chunks/dep-BaOMuo4I.js:51856:27)
      at async viteTransformMiddleware (file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite@5.4.3_@types+node@22.5.4_sass@1.78.0_terser@5.31.6/node_modules/vite/dist/node/chunks/dep-BaOMuo4I.js:61810:24)


 WARN  [Vue Router warn]: No match found for location with path "/_nuxt/components/TipTapImage.vue"


 WARN  [Vue Router warn]: No match found for location with path "/_nuxt/components/TipTapImage.vue"

[21:15:13] ℹ ✨ new dependencies optimized: @tiptap/vue-3, @tiptap/starter-kit, primevue/usetoast, @tiptap/extension-image, lowlight, @tiptap/extension-code-block-lowlight
ℹ ✨ optimized dependencies changed. reloading                                                                                                   21:15:13

 ERROR  Internal server error: Failed to resolve import "E:h5_workspace                                                                           21:15:21
uxt-tiptap-editorsrccustom-extensionsextension-image-upload" from "playground/components/TipTapImage.vue". Does the file exist?
  Plugin: vite:import-analysis
  File: E:/h5_workspace/nuxt-tiptap-editor/playground/components/TipTapImage.vue:6:95
  4  |  import { StarterKit as TiptapStarterKit } from '@tiptap/starter-kit';
  5  |  import { Image as TiptapImage } from '@tiptap/extension-image';
  6  |  import { ImageUpload as TiptapImageUpload, ImagePlaceholder as TiptapImagePlaceholder } from 'E:\h5_workspace\nuxt-tiptap-editor\src\custom-extensions\extension-image-upload';        
     |                                                                                                ^
  7  |  /* Injection by vite-plugin-vue-inspector Start */
  8  |  import { createElementVNode as __createElementVNode,createElementBlock as __createElementBlock,createVNode as __createVNode } from 'vue'
      at TransformPluginContext._formatError (file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite@5.4.3_@types+node@22.5.4_sass@1.78.0_terser@5.31.6/node_modules/vite/dist/node/chunks/dep-BaOMuo4I.js:49192:41)
      at TransformPluginContext.error (file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite@5.4.3_@types+node@22.5.4_sass@1.78.0_terser@5.31.6/node_modules/vite/dist/node/chunks/dep-BaOMuo4I.js:49187:16)
      at normalizeUrl (file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite@5.4.3_@types+node@22.5.4_sass@1.78.0_terser@5.31.6/node_modules/vite/dist/node/chunks/dep-BaOMuo4I.js:63970:23)
      at async file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite@5.4.3_@types+node@22.5.4_sass@1.78.0_terser@5.31.6/node_modules/vite/dist/node/chunks/dep-BaOMuo4I.js:64102:39
      at async Promise.all (index 5)
      at async TransformPluginContext.transform (file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite@5.4.3_@types+node@22.5.4_sass@1.78.0_terser@5.31.6/node_modules/vite/dist/node/chunks/dep-BaOMuo4I.js:64029:7)
      at async file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite-plugin-inspect@0.8.7_@nuxt+kit@3.13.1_magicast@0.3.5_rollup@4.21.2__rollup@4.21.2_vite@5_p6m2mgxqdfrcrveyul5azkmpku/node_modules/vite-plugin-inspect/dist/index.mjs:401:17
      at async PluginContainer.transform (file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite@5.4.3_@types+node@22.5.4_sass@1.78.0_terser@5.31.6/node_modules/vite/dist/node/chunks/dep-BaOMuo4I.js:49033:18)
      at async loadAndTransform (file:///E:/h5_workspace/nuxt-tiptap-editor/node_modules/.pnpm/vite@5.4.3_@types+node@22.5.4_sass@1.78.0_terser@5.31.6/node_modules/vite/dist/node/chunks/dep-BaOMuo4I.js:51856:27)


 ERROR  Pre-transform error: Failed to resolve import "E:h5_workspace                                                                             21:15:21
uxt-tiptap-editorsrccustom-extensionsextension-image-upload" from "playground/components/TipTapImage.vue". Does the file exist?


 WARN  [Vue Router warn]: No match found for location with path "/_nuxt/components/TipTapImage.vue"


 WARN  [Vue Router warn]: No match found for location with path "/_nuxt/components/TipTapImage.vue"
```

**Comment by modbender** [comment]
I was not able to reproduce this bug. Even with same node version. Seems to be Windows specific.
Could you delete lock file and node_modules and retry.

**Comment by dc-wgoossens** [comment]
@modbender I'm having a similar issue on Mac in Docker using Node 22.11.0.

```
Nuxt 3.14.1592 with Nitro 2.10.4                                                                                                                                                                  12:39:58 PM
                                                                                                                                                                                                  12:39:59 PM
  ➜ Local:    http://localhost:3000/
  ➜ Network:  use --host to expose

ℹ Tiptap Editor initialized                                                                                                                                                                      12:40:00 PM
  ➜ DevTools: press Shift + Alt + D in the browser (v1.6.1)                                                                                                                                       12:40:00 PM

ℹ Nuxt Icon server bundle mode is set to local                                                                                                                                                   12:40:00 PM
✔ Vite client built in 29ms                                                                                                                                                                      12:40:01 PM

[12:40:01 PM]  ERROR  Pre-transform error: Failed to resolve import "/code/node_modules/nuxt-tiptap-editor/custom-extensions/extension-image-upload" from "virtual:nuxt:/code/.nuxt/imports.mjs". Does the file exist?

✔ Vite server built in 334ms                                                                                                                                                                     12:40:01 PM
✔ Nuxt Nitro server built in 524 ms                                                                                                                                                        nitro 12:40:02 PM
ℹ Vite client warmed up in 1ms
```

**Comment by modbender** [comment]
@dc-wgoossens could you try with the latest version 2.1.1

**Comment by modbender** [comment]
@dc-wgoossens this should be fixed in the latest version v2.1.4, please check and reply.

**Comment by dc-wgoossens** [comment]
@modbender I can confirm it now works! Thank you!
